### ISSUE TITLE
impossible to import a seed from full node

### ISSUE THREAD

**Issue by ghost** [wavesplatform/WavesGUI#57]
If you call `swagger#!/addresses/seed` in the full node, it returns the already nonce added and hashed value of the walletSeed (and the same seed is stored in the wallet.dat). In the lite client it is impossible to import this seed, if you want to view an address in the lite client that you have created in the full node.

A possible solution might be to check if the seed string is a base58 encoded 32 byte hash, and in this case don't hash it again in buildAccountSeedHash in waves.util.js. There might be other places in the source code which has to be checked for this enhancement and if there are any side effects, but I think this is a minor change.

**Comment by weidisu** [comment]
Yes you can import Waves seed wallet to [WavesLiteWallet ](https://www.waveslitewallet.org) here
